src/rgw/rgw_sal_motr.h

```diff
@@ -921,6 +921,7 @@ class MotrStore : public Store {
     struct m0_realm     uber_realm;
     struct m0_config    conf = {};
     struct m0_idx_dix_config dix_conf = {};
+    uint64_t inline_policy_limit = 2048;
```

Comment:
Per my understanding changes in this file are not reqd

Reply:
done